exampleCourse/questions/element/integerInput/question.html

```diff
@@ -30,49 +50,93 @@
+      <p>
+        The size of the input box can be set with the attribute <code>size</code>, and the attribute
+        <code>show-help-text="false"</code> removes the button for the help popup. This box below
+        has <code>size=5</code> instead of the default <code>size=35</code>.
+      </p>
+      <p>Solve for the value of $c$ given $c = {{params.a}} + {{params.b}}$</p>
     </pl-question-panel>
-    <pl-integer-input answers-name="c_4" display="block" weight="4"></pl-integer-input>
+    <pl-integer-input answers-name="c_5" size="5" show-help-text="false"></pl-integer-input>
   </div>
 </div>
 
 <div class="card my-2">
   <div class="card-body">
-      <pl-question-panel>
-          <p> The size of the input box can be set with the attribute <code>size</code>, and the attribute <code>show-help-text="false"</code> removes the button for the help popup.  This box below has <code>size=5</code> instead of the default <code>size=35</code>. </p>
-      <p> Solve for the value of $c$ given $c = {{params.a}} + {{params.b}}$ </p>
+    <pl-question-panel>
+      <p>
+        This question uses the <code>blank-answer</code> to specify a value to be used if the user
+        leaves it blank.
+      </p>
+      <p>
+        Solve for the value of $c$ given $c = 0^n$ for $n = 1$ (leave the question blank if the
+        value is zero).
+      </p>
     </pl-question-panel>
-    <pl-integer-input answers-name="c_5" size="5" show-help-text="false"></pl-integer-input>
+    <pl-integer-input
+      answers-name="c_6"
+      allow-blank="true"
+      blank-value="0"
+      correct-answer="0"
+    ></pl-integer-input>
   </div>
 </div>
 
 <div class="card my-2">
   <div class="card-body">
     <pl-question-panel>
-      <p>This question uses the <code>blank-answer</code> to specify a value to be used if the user leaves it blank.</p>
-      <p>Solve for the value of $c$ given $c = 0^n$ for $n = 1$ (leave the question blank if the value is zero). </p>
+      <p>The integer can also be expected to be provided in a different base.</p>
+      <p>
+        Solve for the value of $c$ given $c = {{params.a16}}_{16} + {{params.b16}}_{16}$ (provide
+        your answer in base $16$).
+      </p>
     </pl-question-panel>
-    <pl-integer-input answers-name="c_6" allow-blank="true" blank-value="0" correct-answer="0"></pl-integer-input>
+    <pl-integer-input answers-name="c_7" base="16"></pl-integer-input>
   </div>
 </div>
 
 <div class="card my-2">
   <div class="card-body">
     <pl-question-panel>
-      <p>The integer can also be expected to be provided in a different base.</p>
-      <p> Solve for the value of $c$ given $c = {{params.a16}}_{16} + {{params.b16}}_{16}$ (provide your answer in base $16$). </p>
+      <p>
+        The submitted integer can also be very large. This question changes the placeholder text by
+        setting the <code>placeholder</code> attribute.
+      </p>
+      <p>Submit <code>{{params.c_large}}</code>.</p>
```

Comment:
```suggestion
      <p>Submit <code>{{correct_answers.c_large}}</code>.</p>
```
This way you avoid the need for a duplicate param.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/6980/commits/f440f210aacb2dfff8fd45f558ab3770ee08f1d5